lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -367,17 +367,16 @@ protected function _getCreateTableSQL(string $tableName, array $columns, array $
         $options['indexes'] = [];
         $sql                = parent::_getCreateTableSQL($tableName, $columns, $options);
 
-        foreach ($columns as $name => $column) {
+        foreach ($columns as $column) {
             if (isset($column['sequence'])) {
                 $sql[] = $this->getCreateSequenceSQL($column['sequence']);
             }
 
-            if (! isset($column['autoincrement']) || ! $column['autoincrement'] &&
-               (! isset($column['autoinc']) || ! $column['autoinc'])) {
```

Comment:
Not sure if it's worth mentioning the removal of `'autoinc'` as a breaking change. I don't see it mentioned in the documentation and this is the only platform which has it. The rest of the usages were removed prior to DBAL 2.0.

Reply:
I would remove it in another PR and add it to the upgrade doc. That isn't really related to getting rid of the column name index.